backend/README.md

```diff
@@ -1,7 +1,6 @@
 To set up:
-1. Set up a virtual environment.
-2. Activate the virtual environment.
-3. `pip install -r requirements.txt`
+1. install & resolve dependencies & create the virtual environment using the following command: `poetry install`
+3. To run the DEV version add the environment variable: `DEVEL = 'true'`.
 
-To run:
-1. `python __main__.py`
\ No newline at end of file
+To run the server within the virtual environment:
+1. `poetry run python __main__.py`
```

Comment:
Might be worthwhile to add a script to run it: https://python-poetry.org/docs/cli/#run

Example:
```
poetry run server
```

Also not a bad idea to setup boilerplate scripts for stuff like `test` and `lint` which will be useful for CI later.

Reply:
Done, but it did require restructuring the project to the appropriate poetry hierarchy (accomplished by doing `poetry new <project-name>`, then moving the flask code in the appropriate area). Since the server is now inside a package, there had to be some changes in the code as you can see in [8a7a9e1](https://github.com/emseers/eelbot-ui/pull/13/commits/8a7a9e10c60ba6d218bbfe25f9c9dcbf670c66a5).

Reply:
also doing poetry new added a test section, might be useful for future CI

Reply:
Having an extra `backend` folder inside `backend` seems unnecessary. Can the contents inside the child `backend` folder be moved up one level?

Reply:
no, try `poetry new` it will always an extra folder with the same name as the parent.

Reply:
Hmm that's not ideal but not a big deal either. I guess we can live with it.